src/core/blob.cpp

```diff
@@ -14,13 +14,12 @@
 using namespace soci;
 
 blob::blob(session & s)
+	: backEnd_(std::unique_ptr<details::blob_backend>(s.make_blob_backend()))
 {
-    backEnd_ = s.make_blob_backend();
 }
 
 blob::~blob()
 {
-    delete backEnd_;
 }
```

Comment:
I'd leave a comment here just to prevent somebody from removing this "useless" dtor later, and I'd also use `= default`, so:
```cpp
// Note that this dtor must be defined here, where blob_backend class is fully declared.
blob::~blob() = default;
```

Reply:
Yeah defaulting the dtor sounds good :+1: